pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx

```diff
@@ -418,7 +429,7 @@ function SpaceInvites({ spaceId }: { spaceId: SpaceId; }): ReactElement {
 	const addInvite = useCallback((invite: SpaceInvite) => setInvites((inv) => [...inv, invite]), []);
 
 	return (
-		<FieldsetToggle legend='Invites' onChange={ onChange } open={ false }>
+		<FieldsetToggle legend='Invites' onChange={ onChange } open={ true }>
```

Comment:
yes this indeed won't work since the load is tied to the onChange

maybe somehow it could be tied to the Tab onClick so it starts loading as the tab is selected but then the state must be moved up few elements

Reply:
I tried the `onClick` as part of the tab, but then the tab was no longer working. But I also did not know how to move a state up some elements, as I cannot read half of the code from the complexity. *giggles*

Reply:
Jomshir solved the issue.